### ISSUE TITLE
Static in namespace{} in IEX.h 

### ISSUE THREAD

**Issue by Chudleyj** [rodolfobandeira/iex-cpp-client#22]
Why are the functions in the **empty name namespace static**? `g++ *.cpp -std=c++11 -ljsoncpp -lcurl -o exa.out && ./exa.out AMZN -a` compiles fine without the **static** keyword.

**Comment by mpherg** [comment]
https://stackoverflow.com/questions/4725204/use-of-static-variables-and-functions-in-global-scope

**Comment by Chudleyj** [comment]
https://stackoverflow.com/questions/357404/why-are-unnamed-namespaces-used-and-what-are-their-benefits

Empty namespace and static are redundant in this case aren’t they?

**Comment by mpherg** [comment]
Yes, I believe empty namespaces are the C++ preferred way to ensure that a free function is local only to the translation unit.
